dev/NavigationView/NavigationView.xaml

```diff
@@ -113,7 +113,7 @@
                                 <VisualState x:Name="OverflowButtonWithLabel" />
                                 <VisualState x:Name="OverflowButtonNoLabel">
                                     <VisualState.Setters>
-                                        <Setter Target="TopNavOverflowButton.Style" Value="{ThemeResource NavigationViewOverflowButtonNoLabelStyleWhenPaneOnTop}" />
+                                        <Setter Target="TopNavOverflowButton.Style" Value="{ThemeResource TopNavigationViewPaneCustomContentMinWidth}" />
```

Comment:
>TopNavigationViewPaneCustomContentMinWidth [](start = 106, length = 42)

Am i missing something here or is this change setting a style to a double?

Reply:
Not sure how that slipped in, should be fixed now.